src/Mod/Path/PathScripts/PathOpGui.py

```diff
@@ -416,7 +417,34 @@ def __init__(self, obj, features):
         self.panelTitle = 'Base Geometry'
 
     def getForm(self):
-        return FreeCADGui.PySideUic.loadUi(":/panels/PageBaseGeometryEdit.ui")
+        panel = FreeCADGui.PySideUic.loadUi(":/panels/PageBaseGeometryEdit.ui")
+        self.modifyPanel(panel)
+        return panel
+
+    def modifyPanel(self, panel):
+        # Determine if possible operations are available
+        availableOps = list()
+        ops = self.job.Operations.Group
+        for op in ops:
+            if hasattr(op, 'Base') and op.Base.__len__() > 0:
+                availableOps.append(op.Label)
+
+        if availableOps.__len__() > 0:
```

Comment:
Similar, `availableOps` is defined as a list above, so `len(availableOps)` is the most natural way of calling this function.